### ISSUE TITLE
Cypress.env's usage is not clear in the documentation

### ISSUE THREAD

**Issue by BrunoBernardino** [cypress-io/cypress#82]
The example at https://github.com/cypress-io/cypress/wiki/env#no-arguments-usage shows the file like:

```json
// cypress.json
{
  "env": {
    "foo": "bar",
    "baz": "quux"
  }
}
```

But that doesn't work. It should, in fact, be:

```json
// cypress.json
{
  "cypress": {
    "env": {
      "foo": "bar",
      "baz": "quux"
    }
  }
}
```

**Comment by brian-mann** [comment]
You are right. In the next version I'm going to automatically remove the nested `cypress` object from existing `cypress.json`. This has been on my todo list for about 8 months now, so it's time.

**Comment by BrunoBernardino** [comment]
That would actually be better.

I imagine there will be backwards-compatibility so projects don't just start failing :)

**Comment by brian-mann** [comment]
Yes, I'll write it to automatically migrate existing `cypress.json` to the new format so nothing breaks.

**Comment by brian-mann** [comment]
Fixed in [`0.12.4`](https://github.com/cypress-io/cypress/wiki/changelog#0124-11192015).
